### PR TITLE
letters renaming and refactoring cleanup

### PR DESCRIPTION
**Pull request by carolinegeorge-gov**
### Description

Cleaning up db letters schema and instances.

### Checklist

- [x] I have tested the changes locally
- [x] I have updated the documentation if necessary
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [x] I have checked that all unit tests pass by running `npm run test:unit -- run`

### Additional Notes

Just clean up

### CODE REVIEW

File: frontend/app/routes/_gcweb-app.personal-information._index.tsx [hunk 1]
```diff
@@ -87,16 +78,8 @@ export default function PersonalInformationIndex() {
...
-            city={mailingAddressInfo?.city}
-            provinceState={mailingAddressInfo?.province}
-            postalZipCode={mailingAddressInfo?.postalCode}
-            country={mailingAddressInfo?.country} />}
-          {
-            !mailingAddressInfo && <p>{t('personal-information:index.no-address-on-file')}</p>
-          }
-
+          {mailingAddressInfo && <Address address={mailingAddressInfo?.address} city={mailingAddressInfo?.city} provinceState={mailingAddressInfo?.province} postalZipCode={mailingAddressInfo?.postalCode} country={mailingAddressInfo?.country} />}
+          {!mailingAddressInfo && <p>{t('personal-information:index.no-address-on-file')}</p>}
```
Comment (sebastien-comeau) [inline frontend/app/routes/_gcweb-app.personal-information._index.tsx:82]:
```suggestion
          {mailingAddressInfo ? <Address address={mailingAddressInfo.address} city={mailingAddressInfo.city} provinceState={mailingAddressInfo.province} postalZipCode={mailingAddressInfo.postalCode} country={mailingAddressInfo.country} /> : <p>{t('personal-information:index.no-address-on-file')}</p>}
```

File: frontend/app/routes/_gcweb-app.personal-information._index.tsx [hunk 2]
```diff
@@ -66,17 +66,8 @@ export default function PersonalInformationIndex() {
...
-            provinceState={homeAddressInfo?.province}
-            postalZipCode={homeAddressInfo?.postalCode}
-            country={homeAddressInfo?.country} />}
-          {
-            !homeAddressInfo && <p>{t('personal-information:index.no-address-on-file')}</p>
-          }
-
-
+          {homeAddressInfo && <Address address={homeAddressInfo?.address} city={homeAddressInfo?.city} provinceState={homeAddressInfo?.province} postalZipCode={homeAddressInfo?.postalCode} country={homeAddressInfo?.country} />}
+          {!homeAddressInfo && <p>{t('personal-information:index.no-address-on-file')}</p>}
```
Comment (sebastien-comeau) [inline frontend/app/routes/_gcweb-app.personal-information._index.tsx:70]:
```suggestion
          {homeAddressInfo ? <Address address={homeAddressInfo.address} city={homeAddressInfo.city} provinceState={homeAddressInfo.province} postalZipCode={homeAddressInfo.postalCode} country={homeAddressInfo.country} /> : <p>{t('personal-information:index.no-address-on-file')}</p>}
```

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/app/routes/_gcweb-app.personal-information._index.tsx (hunk 1 above)
Comment (carolinegeorge-gov) [reply frontend/app/routes/_gcweb-app.personal-information._index.tsx]: Fixed: 64889e5d0ca6e6e9629cf39b6f55eb8f7b39407e

File: frontend/app/routes/_gcweb-app.personal-information._index.tsx (hunk 2 above)
Comment (carolinegeorge-gov) [reply frontend/app/routes/_gcweb-app.personal-information._index.tsx]: Fixed: 64889e5d0ca6e6e9629cf39b6f55eb8f7b39407e

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**